### PR TITLE
updated Task macro for Vn{ZDC} in Run-2

### PR DESCRIPTION
**Pull request by rihanphys**
Updated the task macros AddTaskVnZDC.C, AliAnalysisTaskVnZDC.cxx AliAnalysisTaskVnZDC.h for run-2.

### CODE REVIEW

**Comment by jmargutt** [comment]
+test

File: PWG/FLOW/Tasks/AliAnalysisTaskVnZDC.cxx [hunk 1]
```diff
@@ -111,35 +111,80 @@ fHist_Cent_count2(NULL),
...
-    fHist_znAx_V0_Cent[i] = NULL;
-    fHist_znAy_V0_Cent[i] = NULL;
+ for(int i=0;i<90;i++){
 
-    fHist_znCx_V0_VxVy[i] = NULL;
-    fHist_znCy_V0_VxVy[i] = NULL;
-    fHist_znAx_V0_VxVy[i] = NULL;
-    fHist_znAy_V0_VxVy[i] = NULL;
+    for(int j=0;j<10;j++){
+    fHist_znCx_V0_VxVy[i][j] = NULL;
```
Comment (dberzano) [inline PWG/FLOW/Tasks/AliAnalysisTaskVnZDC.cxx]: Can you fix the indentation here?

**Comment by jmargutt** [comment]
+1

File: PWG/FLOW/Tasks/AliAnalysisTaskVnZDC.cxx [hunk 2]
```diff
@@ -151,19 +196,31 @@ vzBin(0)
       VzCut[i] = 0;
  }
 
-    DefineInput(1, AliFlowEventSimple::Class()); // Input slot #0 works with an AliFlowEventSimple
+ for(int i=0;i<4;i++){
+  for(int j=0;j<5;j++){
+   fHist_X_vs_Obs_before[i][j] = NULL;
```
Comment (dberzano) [inline PWG/FLOW/Tasks/AliAnalysisTaskVnZDC.cxx]: ...and here

File: PWG/FLOW/Tasks/AliAnalysisTaskVnZDC.cxx [hunk 3]
```diff
@@ -151,19 +196,31 @@ vzBin(0)
...
+ }
 
-    DefineOutput(1,TList::Class());
 
- if(usePhiWeights) {
-    DefineInput(2, TList::Class());              // Input slot #2 is needed for the weights input file
-   }
+
+ for(int i=0;i<10;i++){
+ fFB_Efficiency_Cent[i] = NULL;
```
Comment (dberzano) [inline PWG/FLOW/Tasks/AliAnalysisTaskVnZDC.cxx]: ...and here, and so on...

**Pull request merged**